src/views/iframe.vue

```diff
@@ -40,19 +67,50 @@ export default {
     },
     fullScript() {
       // eslint-disable-next-line no-useless-escape
-      return `<script src="${process.env.VITE_BASE_URL}${this.scriptPath}"><\/script>`
+      return `<script src="${process.env.VITE_BASE_URL}${
+        this.scriptPath
+      }" ${this.options.join(" ")}><\/script>`
```

Comment:
```suggestion
      }" ${this.options.join(" ")}><\\/script>`
```
Pour inclure le backslash et éviter le problème de lint.